spec/interactors/create_temporary_stripe_credit_card_spec.rb

```diff
@@ -38,30 +38,66 @@
 
 
     context "when Stripe customer already associated with the entity" do
+
+      let!(:stripe_customer) { create_stripe_customer(organization: org) }
+
       before do
         order_params[:credit_card][:stripe_tok] = stripe_card_token.id
       end
-      let!(:stripe_customer) {
-        create_stripe_customer(organization: org)
-      }
-
 
-      it "creates a new BankAccount and Stripe::Customer" do
+      it "creates a new BankAccount and Stripe::Customer::Source" do
         result = subject.perform(order_params: order_params, cart: cart, order: order)
         expect(result.success?).to be true
 
         bank_account_id = result.context[:order_params]["credit_card"]["id"]
-        expect(bank_account_id).to be
+        expect(bank_account_id).to_not be_nil
 
         bank_account = BankAccount.find(bank_account_id)
-        expect(bank_account).to be
-        expect(bank_account.bankable).to eq org
-        expect(bank_account.deleted_at).to be
-        expect(bank_account.stripe_id).to be
+        expect(bank_account).to_not be_nil
```

Comment:
I find this easier to read? I try to avoid `to_not` to make it easier to understand

    expect(bank_account).to be

or 

    expect(bank_account).to be_truthy